ui/src/source.ts

```diff
@@ -1,18 +1,18 @@
 import { Readable, writable } from "svelte/store";
```

Comment:
What's the difference between `project/source.ts` and this `source.ts`?

Reply:
`src/screen/project/source` inhabits all the state necessary to drive the Screen Fragment to be found under Project/Source. While `src/source.ts` inhabits all the types and API calls referring to the source domain of the upstream software project.

Reply:
Ah, that's clear :+1: